easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.11-foss-2024a.eb

```
@@ -0,0 +1,3508 @@
+        'checksums': ['5c2daecf6f97aa099ef5fa54f8448518c4f2ed6e44dd29fc60621a70721c60f5'],
+    }),
+    ('ggdist', '3.3.2', {
+        'checksums': ['f6563091cb7ca48b4fb415740a60113be6aac541616f5025110364e12cbe83f1'],
+    }),
+    ('svUnit', '1.0.6', {
+        'checksums': ['263e6c03d6c2717dfd25b96d25f8019c9c98120de280a17224e0d79adba2d522'],
+    }),
+    ('arrayhelpers', '1.1-0', {
+        'checksums': ['5fddd5dd4fb8237bcb24465ef823bc8715ba53e6e5fd7a716c31c48ec128340b'],
+    }),
+    ('tidybayes', '3.0.7', {
+        'checksums': ['1ccc481a430547f0134d32595d59c252a9469a2202cc5300306aa3bb4708290a'],
+    }),
+    ('spdep', '1.3-6', {
+        'checksums': ['eb6d7bbca10698e5527b7963349b694d23f172351832e4ff10629656cf71211a'],
+    }),
+    ('stringmagic', '1.1.2', {
+        'checksums': ['eed1c04aa99be5a99ecfdd32498279f170e261669cc1b91d18673454064b8f8c'],
+    }),
+    ('dreamerr', '1.4.0', {
+        'checksums': ['3e5e4afd10623b6dac6bb9b8bf0480d41c7422884cfec2d9d9786414f9026a87'],
+    }),
+    ('fixest', '0.12.1', {
+        'checksums': ['7ce71ab9856b17000aae7dd578b1b99a43fee35116500626005056b4d9e713df'],
+    }),
+    ('cmna', '1.0.5', {
+        'checksums': ['7cf99880cb70e8fd0b022184167888b1ad32dca503e0250c1d552a84f0613898'],
+    }),
+    ('XBRL', '0.99.19.1', {
+        'checksums': ['ad9ebb5431bdfecc38b8bf3b2552f1a048878a9ac02f5a9d71279b3b099a9757'],
+    }),
+    ('rhandsontable', '0.3.8', {
+        'checksums': ['901ed9c59936f7fa52ad8db3111c8904ab962f9c74f1b6cd40f81683af35d21d'],
+    }),
+    ('missMDA', '1.19', {
+        'checksums': ['f9675884829b2fef75237c335b21991d163674320e766523c71c7a853f95e65c'],
+    }),
+    ('insight', '0.20.5', {
+        'checksums': ['681d0b7ac1f5681e34e6875ac4d43ba15b7bd6931e37a84b999b56e4ece185fd'],
+    }),
+    ('datawizard', '0.13.0', {
+        'checksums': ['bf678185c3b3672df4e21ed30904f6174e15ae5e31ebecd7b232a34c105cfe8a'],
+    }),
+    ('bayestestR', '0.15.0', {
+        'checksums': ['839ed3c76ae5c31b325f604dd2c4b73e361b2f4d0e6a48bec98a9240dc3679f4'],
+    }),
+    ('performance', '0.12.4', {
+        'checksums': ['bee00537b3cb8887fc620b212c35b528058a4da9a61817ff17acd0ad117b5b37'],
+    }),
+    ('fastlogranktest', '0.2.1', {
+        'checksums': ['457c411240b5ee73108042370ccea0f8fa1999e876b2306456d4723be27eb5c8'],
+    }),
+    ('pbmcapply', '1.5.1', {
+        'checksums': ['7ffc2854a384962f0dd523aa9ef33ce8fc290997206b71b840a49049d87112dd'],
+    }),
+    ('PWEALL', '1.3.0.1', {
+        'checksums': ['6fb7eaf557f9bd17bcb94945814dbd77a081d709976bc8046746ff0be7c1dcdc'],
+    }),
+    ('goldilocks', '0.3.0', {
+        'checksums': ['94df7f9d3550dc3790ea3c8f3f364f7131c907a235f498473c8c4e4f959b26c0'],
+    }),
+    ('r2rtf', '1.1.1', {
+        'checksums': ['d64b88187b5eb7bc69eae37f493fdae8417d5127d0a44f1586a00827bc321f52'],
+    }),
+    ('gsDesign', '3.6.5', {
+        'checksums': ['c5a8248708e4803b28cd33189963eb5dd1624df10ecf90f4c8ef7895cc022c0b'],
+    }),
+]
```

```suggestion
    ('gcmr', '1.0.3', {
        'checksums': ['75773d5119179f186467d33608e9ce1c030d0755dfd6ff196546884078e89754'],
    }),
    ('frbs', '3.2-0', {
        'checksums': ['48505d415e3687afcb837e162b63b74173b3cd5df5f96140240e441c86343b2e'],
    }),
]
```

Sync with recent updates to `R-bundle-CRAN-2024.06-foss-2023b.eb`